src/index.js

```diff
@@ -400,6 +400,19 @@ function createConfig(options, entry, format, writeMeta) {
 	const absMain = resolve(options.cwd, getMain({ options, entry, format }));
 	const outputDir = dirname(absMain);
 	const outputEntryFileName = basename(absMain);
+	let tsconfigPath;
+	let tsconfigOptions = {};
+	let ts;
+	if (useTypescript) {
+		tsconfigPath = resolve(
+			options.tsconfig || resolve(options.cwd, 'tsconfig.json'),
+		);
```

Comment:
You don't need this. `rollup-plugin-typescript2` will already try to load `./tsconfig.json` https://www.npmjs.com/package/rollup-plugin-typescript2#plugin-options

Reply:
the approach here is aimed to get the tsconfig json object, then I could use it for rollup output options.
if the interop is enabled, we want to add the __esModule mark for cjs output.

Reply:
Gotcha, that's fair. Sorry, misunderstood.